app/Resources/public/css/print.css

```diff
@@ -4,39 +4,13 @@ body {
     color: #000;
     margin: 0;
     padding: 0;
-    background-color: #fff;
+    font-family: "Arial", serif;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5266914)